### PR TITLE
Config parameters

### PR DESCRIPTION
**Pull request by mfgamallo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Currently supports the following CI servers:
 ## Example config with all possible options
 ```yaml
 title: Example projects
+fetchFrequency: 600000000000 # optional
```
Comment (kszilagyiOT) [inline README.md]: would be nice if this had the unit in the name

File: server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala [hunk 2]
```diff
@@ -17,6 +19,8 @@ object JobConfig {
   implicit val urlUserFormat: YamlFormat[UserRoot] = wrappedYamlString(s => UserRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val urlRestFormat: YamlFormat[RestRoot] = wrappedYamlString(s => RestRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val groupNameFormat: YamlFormat[GroupName] = wrappedYamlString(GroupName.apply)(_.s)
+
+  implicit val finiteDurationFormat: YamlFormat[FiniteDuration] = wrappedYamlNumber(n => FiniteDuration.apply(n.longValue(), NANOSECONDS))(_.toNanos)
```
Comment (kszilagyiOT) [inline server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala]: so what changed here is that it's ns now and not minutes, right? While this is more natural from the point of view of FiniteDuration I don't think I like it. I would have preferred if the config was called `fetchFrequencyMinutes` and it was a number or you can define a `Minutes` case class. Or you could have it called `fetchFrequency` but then it should mandate units in the value, so you could write `fetchFrequency: 60 seconds`.` fetchFrequency: 600000000000` looks quite ugly and not very readable as the user doesn't know the unit.

Comment (mfgamallo) [reply server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala]: Good thinking - done (Minutes case class)

File: server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala [hunk 3]
```diff
@@ -17,7 +19,14 @@ object JobConfig {
...
   implicit val urlRestFormat: YamlFormat[RestRoot] = wrappedYamlString(s => RestRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val groupNameFormat: YamlFormat[GroupName] = wrappedYamlString(GroupName.apply)(_.s)
+
+  implicit val finiteDurationFormat: YamlFormat[FiniteDuration] = wrappedYamlNumber(n => FiniteDuration.apply(n.longValue(), MINUTES))(_.toMinutes)
+}
+
+object Minutes {
+  implicit val format: YamlFormat[Minutes] = yamlFormat1(Minutes.apply)
 }
+final case class Minutes(minutes: FiniteDuration)
```
Comment (kszilagyiOT) [inline server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala]: I think this should have an Int in it, not FiniteDuration. You should remove the FiniteDuration YamlFormat. And you should add a toDuration method to Minutes which converts minute to FiniteDuration

File: README.md [hunk 4]
```diff
@@ -36,6 +36,9 @@ Currently supports the following CI servers:
 ## Example config with all possible options
 ```yaml
 title: Example projects
+fetchFrequency: # optional
```
Comment (kszilagyiOT) [inline README.md]: This is not what I meant exactly but it's also nice, as it's clear that it's minutes, so I am happy with it

File: server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala (hunk 3 above)
Comment (mfgamallo) [reply server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala]: A, makes sense, so FiniteDuration is not involved in the serializing process at all.
Done.

**Pull request merged**